docs/get-started/Velox.md

```diff
@@ -546,6 +546,9 @@ I20231121 10:19:42.348845 90094332 WholeStageResultIterator.cc:220] Native Plan
 ```
 
 
+## Using Stage-Level Resource Adjustment to Avoid OOM(Experimental)
+ see more [here](../VeloxFeature.md)
+
```

Comment:
Can you rename file to VeloxStageResourceAdj.md?

Reply:
Does CK support the feature?

Reply:
> Does CK support the feature?

CK now doesn't support this feature. Theoretically it can work since it's a general feature.